src/composables/useVeBAL.ts

```diff
@@ -96,13 +96,19 @@ export default function useVeBal() {
     balanceFor(networkConfig.addresses.veBAL)
   );
 
+  const hasVeBalBalance = computed(() => Number(veBalBalance.value) > 0);
+
+  const noVeBalBalance = computed(() => veBalBalance.value === '0.0');
```

Comment:
I wanted to use `hasVeBalBalance` instead of this in the LMVoting page but there's a combination of isLoading states between useVeBAL and useBalances that forced me to use this special check.